projects/telemetry/inc/nmea.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <stdbool.h>
```

Comment:
We usually try to have a high-level module description here.

Reply:
Got it, thanks